src/sentry/snuba/tasks.py

```diff
@@ -193,21 +194,36 @@ def _create_in_snuba(subscription):
         snuba_query.environment,
         snuba_query.event_types,
     )
+
+    body = {
+        "project_id": subscription.project_id,
+        "project": subscription.project_id,  # for SnQL SDK
+        "dataset": snuba_query.dataset,
+        "conditions": snuba_filter.conditions,
+        "aggregations": snuba_filter.aggregations,
+        "time_window": snuba_query.time_window,
+        "resolution": snuba_query.resolution,
```

Comment:
How are `time_window` and `resolution` handled here? Does `SubscriptionDataCodec` in snuba know how to parse snql?

Reply:
Ahh, I see we added this here: https://github.com/getsentry/snuba/pull/1866

Reply:
Oh yeah maybe I should have clarified. Snuba has been written to accept subscriptions that have both SnQL and legacy data, although it will only run the queries using the legacy fields at the moment. I have another PR open https://github.com/getsentry/snuba/pull/1910 that will add flags to allow me to test subscriptions using the SnQL data instead of the legacy data.